docs/core/compatibility/sdk/8.0/unlisted-versions.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Breaking change: Unlisted packages not installed by default"
+description: Learn about a breaking change in the .NET 8 SDK where `dotnet tool install` no longer installs tools from unlisted versions of NuGet packages.
+ms.date: 11/07/2023
+---
+# Unlisted packages not installed by default
+
+The [dotnet tool install](../../../tools/dotnet-tool-install.md) commands no longer install tools from [unlisted versions](/nuget/nuget-org/policies/deleting-packages#unlisting-a-package) of NuGet packages by default. You can still force the install by specifying the unlisted version as an exact version surrounded with brackets, for example `--version [5.0.0]`.
+
+## Previous behavior
+
+Previously, when you installed a .NET tool, the .NET SDK installed tools (and versions of tools) without considering whether the tool package was unlisted.
+
+## New behavior
+
+Starting in .NET 8, unlisted tool versions aren't installed unless you specify the exact version using the `--version` option and brackets around the version number. For example, `--version [5.0.0]`.
```

Comment:
Just a note that we're going to fix this quickly to restore the old 'bare' version behavior so when that happens we'll come back and update this breaking change notice to note which version restored the behavior as well.

Reply:
Will it be reverted in .NET 8?

Reply:
yes - we're currently targeting the PR to bring the old format back (while keeping the new, bracketed format) for a .NET 8.0.1xx SDK servicing release: https://github.com/dotnet/sdk/pull/36671

IMO it's still useful to have this notice, because for the whole month after release (at minimum) we'll be in this backwards-incompatible state, and we hope to have quite a lot of activity in that time.